CHANGELOG.md

```diff
@@ -14,6 +14,90 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - For MultiGroupServer, the backend and frontend share each node
 - Moved `MAPL_HistoryGridComp.F90` and related files to `gridcomps/History`
 
+- Major refactoring related to MAPL generic capabilities
+
+    0. Cretaed new subdirectory  "generic"
```

Comment:
Spelling mistake: "Cretaed".

Interestingly, it looks like Markdown does okay with a 0-th led numbered list. Surprising!